tests/test_account_chart.py

```diff
@@ -0,0 +1,122 @@
+    assert modified_accounts['vat_code'].item() == '<values><de>MwSt. 2.6%</de><en>VAT 2.6%</en><fr>TVA 2.6%</fr><it>IVA 2.6%</it></values>'
+    assert modified_accounts['group'].item() == new_account['group']+'/'+new_account['text']
+
+# Test deleting an existent account.
+def test_delete_account():
+    cashctrl_ledger = CashCtrlLedger()
+    cashctrl_ledger.delete_account(account=7777)
+    updated_accounts = cashctrl_ledger.account_chart()
+    assert 7777 not in updated_accounts.index
+
+# Test deleting a non existent account should raise an error.
+def test_delete_non_existent_account_raise_error():
+    cashctrl_ledger = CashCtrlLedger()
+    with pytest.raises(ValueError):
+        cashctrl_ledger.delete_account(7777)
+
+# Test updating a non existent account should raise an error.
+def test_update_non_existent_account_raise_error():
+    cashctrl_ledger = CashCtrlLedger()
+    with pytest.raises(ValueError):
+        cashctrl_ledger.update_account(account=7777, currency='CHF',
+            text='test create account', vat_code='MwSt. 2.6%', group='/Anlagevermögen'
+        )
+
+# Test adding an account without currency should raise an error.
+def test_update_account_without_currency_error():
+    cashctrl_ledger = CashCtrlLedger()
+    with pytest.raises(ValueError):
+        cashctrl_ledger.update_account(account=7777, currency='',
+            text='test create account', vat_code='MwSt. 2.6%', group='/Anlagevermögen'
+        )
+
+# Test updating an account with not valid VAT code should raise an error.
+def test_update_account_with_not_valid_vat_raise_error():
+    cashctrl_ledger = CashCtrlLedger()
+    with pytest.raises(ValueError):
+        cashctrl_ledger.update_account(account=7777, currency='USD',
+            text='test create account', vat_code='Non-Existing Tax Code', group='/Anlagevermögen'
+        )
+
+# Test updating an account without VAT code should raise an error.
+def test_update_account_without_vat_raise_error():
+    cashctrl_ledger = CashCtrlLedger()
+    with pytest.raises(ValueError):
+        cashctrl_ledger.update_account(account=7777, currency='USD',
+            text='test create account', vat_code='', group='/Anlagevermögen'
+        )
+
+# Test updating an account without group should raise an error.
+def test_update_account_with_not_valid_group_raise_error():
+    cashctrl_ledger = CashCtrlLedger()
+    with pytest.raises(ValueError):
+        cashctrl_ledger.update_account(account=7777, currency='USD',
+            text='test create account', vat_code='MwSt. 2.6%', group='/ABC'
+        )
+
+# Test updating an account without group should raise an error.
+def test_update_account_without_group_raise_error():
+    cashctrl_ledger = CashCtrlLedger()
+    with pytest.raises(ValueError):
+        cashctrl_ledger.update_account(account=7777, currency='USD',
+            text='test create account', vat_code='MwSt. 2.6%', group=''
+        )
```

Comment:
Updating an account without group should **not** raise an error. 

To specify 'without group', we pass `group=None`.

`group=''` means that we want an account with a group named as an empty string (which is by definition invalid)

Reply:
Updating an account without group should ** **not** ** raise an error.